### PR TITLE
Fix broken buckets (migration) + Auto-updater fixes

### PR DESCRIPTION
**Pull request by Limezy**
Thanks @fflorent and @tituspijean for your contributions !

### CODE REVIEW

**Comment by Limezy** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -153,9 +148,39 @@ ynh_systemd_action --service_name=nginx --action=reload
...
+			mv "$d" "DATA_$d"
+			pushd $mc_path
+				ynh_exec_warn_less sudo -u $app ./mc mb minio/"$d"
+				ynh_exec_warn_less sudo -u $app ./mc mirror --preserve "$datadir/DATA_$d" minio/"$d"
+				# This is a hack, but it will make outline_ynh users' lifes much easier !
+				if [ "$d" == "outlinestorage" ]
+				then
+					ynh_exec_warn_less sudo -u $app ./mc policy set public minio/outlinestorage
+				else
+					ynh_script_progression --message="Bucket is migrated and objects are now mirrored. However, make sure to set properly access policy of bucket $d. That part of the migration can't be automated, sorry ! You can log into the minio console to perform your changes."
```
Comment (fflorent) [inline scripts/upgrade]: Maybe worth to send an email to the user?

This helper can be useful for that: https://github.com/YunoHost-Apps/Experimental_helpers/blob/master/ynh_send_readme_to_admin/ynh_send_readme_to_admin__2

**Review by fflorent — APPROVED** [review]
LGTM

File: scripts/upgrade (hunk 1 above)
Comment (Limezy) [reply scripts/upgrade]: Maybe, but I'm just trying not to spend  days on that app... I'll try to have a look.
Do you use, or know people using MinIO outside of the outline_ynh case ?

Comment (fflorent) [reply scripts/upgrade]: I understand :)

I don't know. I use it for replicating data from another site.

**Comment by Limezy** [comment]
!testme

**Pull request merged**